demos/APS Monte Carlo Keplerian Covariance Example.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
nit: is the cell output relevant or should it be cleared before submitting the notebook?

Reply:
I thought we submitted the output. We should check on whether that was a mistake in the one I was using or intentional. I liked it because it let me see what I should be seeing if I didn't run it.